curriculum/challenges/chinese/01-responsive-web-design/applied-visual-design/add-a-box-shadow-to-a-card-like-element.chinese.md

```diff
@@ -88,7 +108,10 @@ tests:
 ## Solution
 <section id='solution'>
 
-```js
+
+```html
 // solution required
```

Comment:
It should be `<!-- solution required -->`

Reply:
I am not sure,  because of its transfer from the English version, so maybe keep it is a good idea.